tests/acceptance-tests/test_new_display_configuration.cpp

```diff
@@ -552,6 +552,9 @@ TEST_P(DisplaySubpixelSetting, can_get_all_subpixel_arrangements)
 INSTANTIATE_TEST_SUITE_P(DisplayConfiguration, DisplayPowerSetting,
     Values(mir_power_mode_on, mir_power_mode_standby, mir_power_mode_suspend, mir_power_mode_off));
 
+INSTANTIATE_TEST_SUITE_P(DisplayConfiguration, DisplayOrientationSetting,
+    Values(mir_orientation_normal, mir_orientation_left, mir_orientation_inverted, mir_orientation_right));
```

Comment:
How was this related? I don't see it directly using the removed stuff.

Reply:
It also failed (at runtime) on sid.